### PR TITLE
[Check-in] Feature Branch for tagalog support

### PR DESCRIPTION
**Pull request by brdunfield**
## Description
- Adds a new translation.json for tl
- Adds a new feature toggle: `check_in_experience_translation_disclaimer_tagalog_enabled` to show or hide the 'some content may be in english' disclaimer when the app is in tagalog.
- Adds new tests

Note / question - it looks like the VADS components don't support tagalog

## Original issue(s)
department-of-veterans-affairs/va.gov-team#42789

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by brianseek — APPROVED** [review]
LGTM! On the VADS components, I think we will need to provide them with the initial translations. The same ones we initially got for Spanish.

**Review by theodur — APPROVED** [review]

**Review by it-harrison — APPROVED** [review]

**Review by wullaski — APPROVED** [review]
LGTM! Weird that cypress test 7 timed out but the stress test didn't 🙃

File: src/applications/check-in/locales/tl/translation.json [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+  "your-date-of-birth-can-not-be-in-the-future": "Ang iyong petsa ng kapanganakan ay hindi maaaring nasa hinaharap",
+  "your-last-name": "Ang iyong apelyido",
+  "your-provider-will-call-you": "Tatawagan ka ng iyong provider.",
+  "your-provider-will-call-you-at-your-appointment-time-you-may-need-to-wait-about-15-minutes-for-their-call-thanks-for-your-patience": "Tatawagan ka ng iyong provider sa oras ng iyong appointment. Maaaring kailanganin mong maghintay ng humigit-kumulang 15 minuto para sa kanilang tawag. Salamat sa iyong pagtitiyaga.",
+  "youre-also-responsible-for-protecting-your-personal-health-information-if-you-print-or-download-your-information-or-share-it-electronically-with-others-youll-need-to-take-steps-to-protect-it": "Ikaw rin ay responsible sa pagprotekta sa iyong personal na impormasyong pangkalusugan. Kung ipi-print o ida-download mo ang iyong impormasyon—o ise-share sa iba sa elektronikong paraan—kakailanganin mong gumawa ng mga hakbang para maprotektohan ito.",
+  "youre-checked-in-for-your-appointment": "Naka-check in ka na para sa iyong appointment sa {{date, time}}",
```
Comment (wullaski) [inline src/applications/check-in/locales/tl/translation.json:206]: This key isn't being used any more and the confirmation title is defaulting to English. More of these could be outdated.

**Review by wullaski — COMMENTED** [review]
I noticed an outdated key in the Tagalog json file.

**Pull request merged**